### PR TITLE
MiniZn integration with python

### PR DESCRIPTION
**Pull request by Manojkl**
This file contain the integration of the miniZn output to the python environment.

### CODE REVIEW

**Comment by AshAswin** [comment]
Please ensure that the code works with the updated Master branch setup. It does not work.

File: MiniZn_integration_with_python.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from minizinc import Instance, Model, Solver
+
+# Load platfroms model from file
+platforms = Model("./platforms.mzn")
```
Comment (AshAswin) [inline MiniZn_integration_with_python.py:4]: Update the model path in accordance with the existing "master" branch's file setup.

**Review by AshAswin — CHANGES_REQUESTED** [review]
Please ensure that the code works in the existing master branch's file setup.

File: MiniZn_integration_with_python.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# # Create an Instance of the platforms model for Gecode
+instance = Instance(gecode, platforms)
+# Assign 4 to n
+instance["name"] = 1
+instance["min_force_sensor_count"] = 1
+instance["min_tactile_sensor_count"] = 100
+instance["min_memory_fused"] = 400
+instance["np"] = 4
+instance["platforms"] = set([1,2,3,4])
+# solve to get the genral solution
```
Comment (AshAswin) [inline MiniZn_integration_with_python.py]: Be mindful of typos in your comments

**Review by AshAswin — CHANGES_REQUESTED** [review]

**Comment by AshAswin** [comment]
Use lower snake case for file names. Provide a concise name for the python file

File: MiniZn_integration_with_python.py (hunk 1 above)
Comment (Manojkl) [reply MiniZn_integration_with_python.py:4]: Please call appropriate file inside the plaforms.mzn to make the python integration work.

File: MiniZn_integration_with_python.py (hunk 2 above)
Comment (Manojkl) [reply MiniZn_integration_with_python.py]: All changes are updated

**Pull request merged**